eav/utils.py

```diff
@@ -0,0 +1,21 @@
+import sys
+from django.db.models import Q
+
+
+def print_q_expr(expr, indent="", is_tail=True):
+    '''
+    Simple print method for debugging Q-expressions' trees.
+    '''
+    sys.stdout.write(indent)
```

Comment:
@IwoHerka as above - or is it intentional to leave the empty line in some cases?

Reply:
Accepted convention is to leave one empty line in class/module docstring and no empty space in case of functions. Therefore, I will ensure the rest of the new code adheres to this rule.

Reply:
👍